flask_debugtoolbar/static/css/toolbar.css

```diff
@@ -167,6 +167,7 @@
   left:0px;
   background-color:#eee;
   color:#666;
+  overflow:auto;
```

Comment:
Why? Is this related to the success codes change?

Reply:
Nope, it's unrelated - my apologies! Our various commits should have been in different branches, not stacked onto the `master` branch of the cloned repository. This one line was to allow scrolling of panels that overflowed beyond the vertical space in the browser. Go ahead and reject this if you'd like and we can put in a separate PR.